packages/components/tabs/src/tab-group.ts

```diff
@@ -32,380 +30,351 @@ const OBSERVER_OPTIONS: MutationObserverInit = {
+      // container or wrapper. The tablist is the element that
+      // changes size for example when fonts are loaded. The
+      // other elements do not change size while the tablist does.
+      this.#resizeObserver.observe(tablist);
+    });
   }
 
-  override render(): TemplateResult {
-    this.#moreButton = this.renderRoot.querySelector('#more-btn') as HTMLButtonElement;
-    this.#allTabs = Array.from(this.querySelectorAll('sl-tab'))?.map(tab => tab.cloneNode(true)) as Tab[];
-    this.#allTabs.forEach(tab => tab.classList.add('listbox-tab'));
+  override disconnectedCallback(): void {
+    this.#resizeObserver.disconnect();
+    this.#mutationObserver.disconnect();
 
+    super.disconnectedCallback();
+  }
+
+  override updated(changes: PropertyValues<this>): void {
+    super.updated(changes);
+
+    if (changes.has('alignTabs')) {
+      this.#shouldAnimate = false;
+      this.#updateSelectionIndicator();
+      this.#shouldAnimate = true;
+    }
+  }
+
+  override render(): TemplateResult {
     return html`
-      <div class="container" part="container" @keydown=${this.#handleKeydown}>
-        <div class="wrapper" part="wrapper">
-          <div @click=${this.#handleTabChange} role="tablist" part="tab-list">
-            <span class="indicator" role="presentation"></span>
-            <slot name="tabs" @slotchange=${() => this.#rovingTabindexController.clearElementCache()}></slot>
-            <div
-              ${anchor({ element: this.#moreButton, position: this.vertical ? 'bottom-start' : 'bottom-end' })}
-              @click=${this.#handleTabChange}
-              @toggle=${this.#onToggle}
-              aria-labelledby="more-btn"
-              id="tabs-popover"
-              part="listbox"
-              popover
-              role="listbox"
-            >
-              ${this.#allTabs}
+      <div part="container">
+        <div part="wrapper">
+          <div class="fade-container">
+            <div class="fade fade-left"></div>
+            <div class="fade fade-right"></div>
+            <div @scroll=${this.#onScroll} part="scroller">
+              <div @click=${this.#onClick} @keydown=${this.#onKeydown} part="tablist" role="tablist">
+                <span class="indicator" role="presentation"></span>
+                <slot @slotchange=${this.#onTabSlotchange} name="tabs"></slot>
+              </div>
             </div>
           </div>
+
+          ${this.showMenu
+            ? html`
+                <sl-menu-button aria-label=${msg('Show all')} fill="ghost">
+                  <sl-icon name="ellipsis" slot="button"></sl-icon>
+                  ${this.menuItems?.map(
+                    menuItem => html`
+                      <sl-menu-item @click=${() => this.#onMenuItemClick(menuItem.tab)} ?disabled=${menuItem.disabled}>
+                        <div class="title">${menuItem.title}</div>
+                        ${menuItem.subtitle ? html`<div class="subtitle">${menuItem.subtitle}</div>` : nothing}
```

Comment:
According to previous design and Roald's opinion, can you please remove subtitles from dropdown? So we will always need to have titles only in the menu.